### PR TITLE
[Proposal/Experimental] :recycle: I implemented the timetable droid-kun to blink.

### PR DESCRIPTION
**Pull request by Corvus400**
## Issue
- None.

## Overview (Required)
- Added animation of Droid-kun blinking his eyes.
- I really wanted the sweat to flow as well, but it was impossible because the AVD file creation service we used made the color of the sweat look wrong.

## Movie (Optional)
Light | Dark
:--: | :--:
<video src="https://github.com/DroidKaigi/conference-app-2023/assets/13657682/33c5621d-7311-47b3-8e05-a0b4e6c90e6b" width="300" > | <video src="https://github.com/DroidKaigi/conference-app-2023/assets/13657682/85f24fc2-48a6-4cb4-900a-ef2d74c583a5" width="300" >

### CODE REVIEW

File: feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/section/TimetableHeader.kt [hunk 1]
```diff
@@ -31,10 +43,30 @@ fun TimetableHeader(modifier: Modifier = Modifier) {
...
-        Image(
-            modifier = Modifier.size(width = 185.dp, height = 169.dp),
-            painter = painterResource(id = R.drawable.img_droid_kun_in_bath),
-            contentDescription = null,
-        )
+        Box {
+            var atEnd by remember { mutableStateOf(false) }
+            Image(
+                modifier = Modifier.size(width = 185.dp, height = 169.dp),
+                painter = painterResource(id = R.drawable.img_droid_kun_in_bath),
```
Comment (takahirom) [inline feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/section/TimetableHeader.kt:50]: Thanks for amazing animation. How about using png or webp for img_droid_kun_in_bath to improve performance? I'm not sure but the warning will occur if the vector drawable is big.

**Comment by takahirom** [comment]
I got designer's feedback. Could you improve it?

> 目を閉じている時間は短めにして欲しいです！イメージは↓の感じです...
目開きっぱなし約2秒→瞬き2回(ループ)
(理由:目を閉じた時の目の太さは口の線の太さと同じぐらいになるが、今の実装だと目が細くなるので違和感ありです :おじぎ::肌色-2:...)

**Comment by Corvus400** [comment]
@takahirom 
I modified it like this, how do you like it?

https://github.com/DroidKaigi/conference-app-2023/assets/13657682/02a8d3e9-1ea9-4c4f-aa0f-ba91063b08e1

**Review by takahirom — APPROVED** [review]
Looking great!

**Pull request merged**